Index: src/org/eclipse/php/internal/core/ast/nodes/FormalParameter.java
===================================================================
RCS file: /cvsroot/tools/org.eclipse.pdt/plugins/org.eclipse.php.core/src/org/eclipse/php/internal/core/ast/nodes/FormalParameter.java,v
retrieving revision 1.17
diff -u -r1.17 FormalParameter.java
--- src/org/eclipse/php/internal/core/ast/nodes/FormalParameter.java	24 Apr 2009 19:58:44 -0000	1.17
+++ src/org/eclipse/php/internal/core/ast/nodes/FormalParameter.java	23 May 2009 11:02:31 -0000
@@ -194,7 +194,7 @@
 	 * @return default value of this parameter
 	 */
 	public Expression getDefaultValue() {
-		if (!this.defaultInitialized) {
+		/*if (!this.defaultInitialized) {
 			// lazy init must be thread-safe for readers
 			synchronized (this) {
 				if (!this.defaultInitialized) {
@@ -205,7 +205,7 @@
 					postLazyInit(defaultValue, DEFAULT_VALUE_PROPERTY);
 				}
 			}
-		}
+		}*/
 		return defaultValue;
 	}
 
@@ -281,7 +281,7 @@
 	 * @return the type of this parameter
 	 */
 	public Expression getParameterType() {
-		if (!this.parameterTypeInitialized) {
+		/*if (!this.parameterTypeInitialized) {
 			// lazy init must be thread-safe for readers
 			synchronized (this) {
 				if (!this.parameterTypeInitialized) {
@@ -292,7 +292,7 @@
 					postLazyInit(parameterType, PARAMETER_TYPE_PROPERTY);
 				}
 			}
-		}
+		}*/
 		return parameterType;
 	}
 
